book.js

```diff
@@ -1,52 +1,68 @@
-const books = JSON.parse(localStorage.getItem('books')) || [];
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = Date.now();
+  }
```

Comment:
You did a good job working on this milestone! 🤓 However, I have some observations and thoughts about your code:

- [ ] You named your class after the `Book` object with author, id, and title. However, this class also has methods like `getAllBooks`, `addBook`, and so on. Those methods don't have a lot of sense considering that a class is supposed to describe a blueprint for a real object and adding a book into a `Book` instance doesn't make much sense. 🤔 What I suggest is that instead of describing the `Book` class, you describe the whole list of books, like this:

```js
class Books {
  constructor() {
    this.books = JSON.parse(localStorage.getItem('books')) || [];
  }

  addBook(title, author) {
    // Logic to add a book to the list here
  }

  removeBook(id) {
    // logic to remove a book from the list here
  }

  displayBooks() {
    // logic to display books here
  }

  // getter method
  getBooks() {
    return this.books;
  }
}

```

Implementing this now makes more sense because all the methods affect or use the `books (list)` property of the class, and adding, removing, and displaying books from this list are all logical actions in a List instance. 🤖 Please consider applying this! 🤓 🔧